src/main/java/edu/harvard/dbmi/avillach/dictionary/concept/ConceptService.java

```diff
@@ -71,4 +71,11 @@ public Optional<Concept> conceptDetailWithoutAncestors(String dataset, String co
         return getConcept(dataset, conceptPath, false);
     }
 
+    public Optional<List<Concept>> conceptsWithDetail(List<String> conceptPaths) {
+        if (conceptPaths.isEmpty()) {
```

Comment:
Same complaint. I feel like the zero value of an empty list is already acceptably baked into the type. Don't really like the optional here.

Reply:
I can remove it. The optional was because I was trying to do some TDD. I started with returning an optional if the list was empty as my first test. I can remove this now and it should still work.